cnxdb/archive-sql/schema/triggers.sql

```diff
@@ -235,7 +235,7 @@ CREATE OR REPLACE FUNCTION rebake()
 AS $$
 BEGIN
   UPDATE modules SET stateid = 5 
-    WHERE module_ident = NEW.module_ident;
+    WHERE module_ident = NEW.module_ident and stateid not in (5, 6);
```

Comment:
We are weaving a very nice knot. 😜  Let's just hope we don't need to add or remove states.

Reply:
Is it too late to suggest `UPPER_SNAKE_CASE` for states in the DB rather than integers? It seems like that would make the SQL files (and other code) more readable.

Reply:
@philschatz, this is a sql statement, so I'm not sure the suggested variable name as a constant applies.